eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumWallet.scala

```diff
@@ -511,8 +512,8 @@ object ElectrumWallet {
   case object GetBalance extends Request
   case class GetBalanceResponse(confirmed: Satoshi, unconfirmed: Satoshi) extends Response
 
-  case object GetXpub extends Request
-  case class GetXpubResponse(xpub: String, path: String) extends Response
+  case object GetRootPub extends Request
```

Comment:
We should keep the old name. Xpub is widely used even when it's actually a Upub/Zpub etc... and conveys more meaning.